### ISSUE TITLE
Predict using just item_features

### ISSUE THREAD

**Issue by ManishBarnwal** [lyst/lightfm#218]
Hi,

I have the user-item interactions matrix and item_features matrix created for the items viewed as training data. I train the model using:

`model_si = LightFM(loss='bpr', learning_rate=optimal_learning_rate,
                no_components=optimal_no_components, random_state = 2709)
`

 `model_si.fit(X_train, item_features=item_features_concat, epochs=optimal_epochs)`

So, as you can see I have passed the side-information for the items only. Now if I want to predict for 100 users (test data):
            a.  who were not in the training data but I have the list of items (say a total of 500 items) these users have viewed 
            b. I extract the item_features for these 500 items, say test_data_item_features
            c. Remember that I never had the user_features so I don't have this for test-data as well

If I want to predict for user 0 in test-data who has viewed three items (say 0, 1, 2) of the total of 500 items. How do I use `predict` function to get the recommendations for this user? I don't see a way to pass the test-data interactions of this user to the predict function.

Example code: 
`score_si = model_si.predict(user_ids=0, item_ids=np.arange(500), item_features=test_data_item_features)
`
The above code does not take into consideration the interactions they had in the test-data. 

Thank you so much for this package. Looking forward to your reply.

**Comment by maciejkula** [comment]
It's not possible to predict for users (or, more accurately using user features) that are not present in the training set.

It could be done using fold-in, but that's not currently implemented.

**Comment by satya326** [comment]
I have trained my model on some n number of users without any user or item features. suppose i have to predict for user n+1 who has done items 3,4,5 of total 100 items(present in the training set), where will i pass this information while predicting.

**Comment by maciejkula** [comment]
See answers above: this is currently _not_ possible.
